<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>localVaultSetup</class>
 <widget class="QWidget" name="localVaultSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>License</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>The space you offer to the network is also how much space you will have to store your files in. A minimum space of 1 GB = 1024 MB is recommended.

Leaving port as 0 will allow Perpetual Data to determine one for you automatically.

Choose the directory so that you are able to increase the space when you desire.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="labelSpace">
       <property name="text">
        <string>Space to offer (in MB):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineSpace">
       <property name="inputMask">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Port (0 recommended):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="linePort">
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Data directory:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineDirectory"/>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBrowse">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
